### ISSUE TITLE
Support @file on _userdata update

### ISSUE THREAD

**Issue by bchatelard** [scaleway/scaleway-cli#321]

**Comment by QuentinPerez** [comment]
Hi @bchatelard,

@file is already supported,
```console
./scw _userdata 8cc2d32c-5444-4998-81e9-964134700ba7 CLOUD-INIT=@cloud-init.yml
```
```yaml
./scw _userdata 8cc2d32c-5444-4998-81e9-964134700ba7 CLOUD-INIT
#cloud-config

hostname: "coreos"

coreos:
  units:
    - name: runcmd.service
      command: start
      content: |
        [Unit]
        Description=Mark as booted

        [Service]
        Type=oneshot
        ExecStart=/bin/sh -c "curl http://169.254.42.42/state -X PATCH -d '{\"state_detail\": \"booted\"}' -H 'Content-Type: application/json'"
```

**Comment by moul** [comment]
@bchatelard tested with `@~/path/to/file`

**Comment by QuentinPerez** [comment]
Fixed with the PR #332
